x-pack/plugins/security_solution/public/exceptions/utils/pages.utils.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { ChromeBreadcrumb } from '@kbn/core/public';
+import { EXCEPTIONS_PATH } from '../../../common/constants';
+import { SecurityPageName } from '../../app/types';
+import type { GetSecuritySolutionUrl } from '../../common/components/link_to';
+import type { RouteSpyState } from '../../common/utils/route/types';
+
+const isListDetailPage = (pathname: string) =>
+  pathname.includes(EXCEPTIONS_PATH) && pathname.includes('/details');
+
+export const getTrailingBreadcrumbs = (
+  params: RouteSpyState,
+  getSecuritySolutionUrl: GetSecuritySolutionUrl
+): ChromeBreadcrumb[] => {
+  let breadcrumb: ChromeBreadcrumb[] = [];
+
+  if (isListDetailPage(params.pathName) && params.state?.listName) {
+    breadcrumb = [
+      ...breadcrumb,
+      {
+        text: params.state.listName,
+        href: getSecuritySolutionUrl({
+          deepLinkId: SecurityPageName.exceptions,
+          path: '',
+        }),
```

Comment:
NIT: This `href` prop is not strictly correct, it points to the main `/exceptions` path instead of the list detail page. Since this is the last breadcrumb, it is not clickable, the `href` prop is not needed, so we could just remove it by now.

However, if someday we need to append another breadcrumb after it, we will have to define the list detail `href` to make this breadcrumb (not the last one anymore) clickable. To do so, we would need to get the exception list id and generate the `/detail/[id]` path.
The generic way to do it would be to rename the `/:exceptionListId` parameter in the path by the generic `/:detailName` param name, this way we could generate the `detail/${params.detailName}` here.

Reply:
Thanks @semd for the details explanations, done here [b324582](https://github.com/elastic/kibana/pull/145605/commits/b324582c625518ae867d217d4d26c61d49f3fca6)